### ISSUE TITLE
Popup on TV must be disabled

### ISSUE THREAD

**Issue by thebe14** [TeamNewPipe/NewPipe#10382]
### Checklist

- [X] I am able to reproduce the bug with the latest version given here: [CLICK THIS LINK](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.25.2

### Steps to reproduce the bug

1. Install on TV
2. Start a video and click on Popup button below video
3. Grant the NewPipe app the popup permissions in the dialog that opens (I cannot open this dialog from anywhere else on the TV, maybe add a lint to the settings to open it?)
4. Video starts in the popup window
5. Cannot get rid of the popup window

### Expected behavior

No popup playback is possible/allowed on TV.

Please remove this option when running on a TV. It is useless, catastrophically degrades the user experience and it ruins the TV, the TV cannot be used to watch/play anything else until the popup window is taken care of. 

### Actual behavior

The TV remote has NO way of ever navigating to this window or controlling it. Is stays as a black box (after the video from which you started it from finishes playing) in the middle of the TV screen, persists after TV reboot, I had to restore the TV to factory settings to get rid of it.

### Affected Android/Custom ROM version

Android TV OS v11

### Affected device model

Philips 65PUS8506

### Additional information

Security patch level 05.apr.2023
Kernel v4.19.182-ab2717

**Comment by peat80** [comment]
Indeed, an option to disable the popup player would be a good addition.
On tv it probably should be disabled by default.
But the option to manually enable it should be kept.
